docs/source/troubleshooting/osx.rst

```diff
@@ -27,7 +27,7 @@ The colors look weird in iTerm2!
 
 * The arrows may have the wrong colors if you have changed the “minimum 
   contrast” slider in the color tab of your OS X settings.
-* Please disable background transparency to resolve this issue.
+* If you're using transparency, check "Keep background colors opaque"
```

Comment:
Why did you remove dot? Also note what is used as quotes just the line above and as ampersand on line 8.